runtime/doc/develop.txt

```diff
@@ -242,13 +242,14 @@ If the function acts on an object then {thing} is the name of that object
 {thing} is usually omitted (but consider "namespacing" your global operations
 with a {thing} that groups functions under a common concept).
 
-Use existing common {action} names if possible:
-    add   Append to, or insert into, a collection
-    del   Delete a thing (or group of things)
-    exec  Execute code
-    get   Get a thing (or group of things by query)
-    list  Get all things
-    set   Set a thing (or group of things)
+Use existing common {action} names if possible. For example:
```

Comment:
Let's try to keep this list up to date. That removes ambiguity. So we don't need "for example.
```suggestion
Use existing common {action} names if possible:
```